app/views/communicart_mailer/_comments.html.haml

```diff
@@ -0,0 +1,21 @@
+- if comments.any?
+  %table.submodel-section
+    %tr
+      %td.w-container{:colspan => "2"}
+        %strong Comments
+    - comments.each do |c|
+      %tr
+      - if c.update_comment
+        class="comment-update"
+      \&gt;
```

Comment:
Here's an example; unless I misunderstand, this isn't quite right. We don't want the body of the `tr` to be the string `class="comment-update"`; we don't want the greater than symbol; we _want_ the `td`s to be contained in this `tr`.

We want something like
```haml
%tr{class: "comment-update" if c.update_comment}
    %td
    ...
```